### ISSUE TITLE
Error: Command line operation failed with code 1.

### ISSUE THREAD

**Issue by satyamakgec** [OpenZeppelin/solidity-docgen#12]
I am trying to generate the docs using the solidity-docgen. Earlier it was working fine but suddenly it is behaving very weird.
getting this error 

> Error: Command line operation failed with code 1.
 at handleErrorCode (/usr/local/lib/node_modules/solidity-docgen/lib/index.js:133:11)
    at parseProject (/usr/local/lib/node_modules/solidity-docgen/lib/index.js:123:3)
    at exports.default (/usr/local/lib/node_modules/solidity-docgen/lib/index.js:20:23)
    at Command.program.version.arguments.option.action (/usr/local/lib/node_modules/solidity-docgen/bin/solidity-docgen:32:5)
    at Command.listener (/usr/local/lib/node_modules/solidity-docgen/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:180:13)
    at Command.parseArgs (/usr/local/lib/node_modules/solidity-docgen/node_modules/commander/index.js:653:12)
    at Command.parse (/usr/local/lib/node_modules/solidity-docgen/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/usr/local/lib/node_modules/solidity-docgen/bin/solidity-docgen:44:9)
    at Module._compile (internal/modules/cjs/loader.js:654:30)

**Comment by spalladino** [comment]
@satyamakgec the error seems to be happenning as part of `parseProject`, which invokes the solidity compiler. Make sure you have a working version of solc (with version greater than 0.4.21), and that your set of contracts do compile ok. 

I'm afraid that info on errors is not very clear atm.

**Comment by satyamakgec** [comment]
@spalladino Thanks for the prompt answer. Yes, I do have the solc version greater than the 0.4.21. My version is 0.4.24.  
My contracts are compiling ok but some of the warnings occurred because of compiler version My contract suite have the `oraclize.sol` which was built on `0.4.18` solc version. But in a whole I am using 0.4.24 solc compiler. 
is oraclize.sol will be the cause of the failure? I tried after removing the oraclize and its dependent contract but still, I am getting the same error.  Is there any way to ignore those contracts? I tried to use the `-excludes` flag but it seems it generates a document for them.
